scripts/mint-nft.js

```diff
@@ -0,0 +1,93 @@
+require("dotenv").config()
+const API_URL = process.env.API_URL
+const PUBLIC_KEY = process.env.PUBLIC_KEY
+const PRIVATE_KEY = process.env.PRIVATE_KEY
+
+const { createAlchemyWeb3 } = require("@alch/alchemy-web3")
+const web3 = createAlchemyWeb3(API_URL)
+
+const contract = require("../artifacts/contracts/MyNFT.sol/MyNFT.json")
+const contractAddress = "0x18dec4D7dBCAb4c4d25aFa463a09d781C49b5aaa"
+const nftContract = new web3.eth.Contract(contract.abi, contractAddress)
+
+async function mintNFT(tokenURI) {
+    const nonce = await web3.eth.getTransactionCount(PUBLIC_KEY, "latest") //get latest nonce
+
+    //the transaction
+    const tx = {
+        from: PUBLIC_KEY,
+        to: contractAddress,
+        nonce: nonce,
+        gas: 500000,
+        data: nftContract.methods.mintNFT(PUBLIC_KEY, tokenURI).encodeABI(),
+    }
+
+    const signPromise = web3.eth.accounts.signTransaction(tx, PRIVATE_KEY)
+    signPromise
+        .then((signedTx) => {
+            web3.eth.sendSignedTransaction(
+                signedTx.rawTransaction,
+                function (err, hash) {
+                    if (!err) {
+                        console.log(
+                            "The hash of your transaction is: ",
+                            hash,
+                            "\nCheck Alchemy's Mempool to view the status of your transaction!"
+                        )
+                    } else {
+                        console.log(
+                            "Something went wrong when submitting your transaction:",
+                            err
+                        )
+                    }
+                }
+            )
+        })
+        .catch((err) => {
+            console.log(" Promise failed:", err)
+        })
+}
+
+async function sendToken() {
+    const nonce = await web3.eth.getTransactionCount(PUBLIC_KEY, "latest") //get latest nonce
+
+    //the transaction
+    const tx = {
+        from: PUBLIC_KEY,
+        to: contractAddress,
+        nonce: nonce,
+        gas: 500000,
+        data: nftContract.methods.safeTransferFrom(PUBLIC_KEY, "0x7aaa2785baaf248b91fbc64b70280bcf3ddd4e09", "1").encodeABI(),
```

Comment:
`0x7aaa2785baaf248b91fbc64b70280bcf3ddd4e09` is wallet ID? Want to make it a const?